mods/ra/maps/survival01/survival01.lua

```diff
@@ -302,9 +302,9 @@ InitObjectives = function()
 	end)
 
 	SurviveObj = allies.AddPrimaryObjective("Enforce your position and hold-out the onslaught\nuntil reinforcements arrive.")
-	KillSams = allies.AddSecondaryObjective("Destroy the two SAM Sites before reinforcements\narrive.")
-	Media.DisplayMessage("The soviets are blocking our GPS. We need to investigate their new technology.")
-	CaptureAirfields = allies.AddSecondaryObjective("Capture and hold the soviet airbase\nin the north east.")
+	KillSams = allies.AddSecondaryObjective("Destroy the two SAM sites before reinforcements\narrive.")
```

Comment:
like with the refinery, I suspect that is because it's kind of a heading in the tooltip:
https://en.wikipedia.org/wiki/List_of_Nike_missile_sites
http://cnc.wikia.com/wiki/SAM_site_%28Red_Alert%29 puts it in lowercase